sql/testdata/select

```diff
@@ -66,7 +66,7 @@ statement ok
 INSERT INTO abc VALUES (0, 1, 2)
 
 query II
-SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b/a > 1.5 ELSE false END
+SELECT a,b FROM abc WHERE CASE WHEN a != 0 THEN b::float/a::float > 1.5 ELSE false END
```

Comment:
this is exactly why we originally decided against integer division. I'm :-1: on this change.